eng/unsetispackable.targets

```diff
@@ -1,8 +1,9 @@
 <Project TreatAsLocalProperty="IsPackable">
 
-  <!-- Unset global IsPackable property. -->
+  <!-- Unset global IsPackable property.
```

Comment:
The word unset no longer applies. Here and in the filename. I think they need to be changed to avoid confusion in the future.

And good catch, I guess that's the behavior I was seeing in my PR: empty being treated as true. But it didn't occur to me that the right thing to do was to set this to false. Unsetting the property sounded like the actual behavior we wanted.

Reply:
The problem here is that if it's unset, NuGet's packaging targets again set it to true: https://github.com/NuGet/NuGet.Client/blob/8fac9ebc758a0edae31c5f4c8fe2938b81f109dc/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets#L36